src/Pages/Register.jsx

```diff
@@ -0,0 +1,141 @@
+import { faLocationDot, faLock, faUser } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { useState } from "react";
+import { Button, Col, Form, InputGroup, Row } from "react-bootstrap";
+
+import './Register.css'
+
+
+
+function Register(){
+  const [form , setForm]=useState({
+    userName: '', // Default value for username field
+    passward: '', // Default value for password field
+    confirmPassward: '', // Default value for confirm password field
+  });
+  const [errors , setErrors]=useState({
+    userName: '', // Initialize with an empty string for default value
+    passward: '', // Initialize with an empty string for default value
+    confirmPassward: '', // Initialize with an empty string for default value
+  });
+
+  const setField =(field , value)=>{
+    setForm({
+      ...form,
+      [field]:value
+    })
+  }
+  const validateForm = () =>{
+    // eslint-disable-next-line no-unused-vars
+    const {userName, passward, confirmPassward}= form
+    const newErrors = {}
+
+    if(!userName || userName==='') {
+    newErrors.userName = 'please enter your username'
+    }
+
+    if (passward !== confirmPassward) {
+      newErrors.confirmPassward = 'Password does not match';
+    }
+
+    setErrors(newErrors);
+    return(newErrors);
+  }
+  const handleSubmit = (e) => {
+    e.preventDefault();
```

Comment:
worth noticing when using preventDefault()

when using e.preventDefault() in events in javascript, it will disable any event for the children components in some component.
 very simple function we usually use but it has a huge effect.
https://www.w3schools.com/jsref/event_preventdefault.asp#:~:text=The%20preventDefault()%20method%20cancels,link%20from%20following%20the%20URL